cmd/admin-trace.go

```diff
@@ -94,6 +97,18 @@ var adminTraceFlags = []cli.Flag{
 		Name:  "filter-size",
 		Usage: "filter size, use with filter (see UNITS)",
 	},
+	cli.StringFlag{
+		Name:  "rotate-trace-logs-size",
+		Usage: "maximum size of a rotated trace log file (see UNITS)",
+	},
+	cli.StringFlag{
+		Name:  "rotate-trace-logs-frequency",
+		Usage: "frequency, in terms of seconds, of trace log file rotations",
+	},
+	cli.StringFlag{
+		Name:  "rotate-trace-logs-directory",
```

Comment:
How about having the be a pattern?

It could be run though [time.Format](https://pkg.go.dev/time#Time.Format) [layout](https://pkg.go.dev/time#pkg-constants) to add times, so you could do `/tmp/2006-01-02/minio-log-15-04-05.log`, so logs can be placed in separate folders?

Reply:
Added --rotate-trace-logs-pattern flag to place logs in time-striped folders, using time.Format